src/geo/map/plain-layer.js

```diff
@@ -11,6 +14,28 @@ var PlainLayer = LayerModelBase.extend({
     className: 'plain',
     color: '#FFFFFF',
     image: ''
+  },
+
+  initialize: function (attrs, options) {
+    attrs = attrs || {};
+    options = options || {};
+    if (!options.vis) throw new Error('vis is required');
+
+    this._vis = options.vis;
+
+    this.bind('change', this._onAttributeChanged, this);
```

Comment:
take into account that, for views, if you have a custom model (not this.model), you need to keep using `.add_related_model(whatevermodel)`. But with `listenTo`, you don't need that.